src/plugins/ai_assistant_management/selection/server/plugin.ts

```diff
@@ -79,6 +86,66 @@ export class AIAssistantManagementSelectionPlugin
       },
     });
 
+    core.capabilities.registerProvider(() => {
```

Comment:
__note:__ forget to comment on this one yesterday - I don't think we need this here - I believe it's only needed when the feature/plugin can be used with Elastic Stack security disabled which is unlikely to be the case for this plugin.